### PR TITLE
Add GeoCoding to Location

### PR DESCRIPTION
**Pull request by Tinusw**
## Ch, Ch, Ch, Ch, Changes...

GeoCoder now takes addresses and converts them to latitude and longitude!

geocoder_example

## Notes
GeoCoder is actually pretty smart and tries to deduce a latitude & longitude based on just about any descriptive input (e.g. typing Te Papa will also work). Fields can be filled out or left blank without compromising functionality. We may have to make this clearer to the user in the UI eventually.

## Deploy actions

None

## Checklist
Make sure these boxes are checked before submitting your contribution

- [X] This PR has only the commits related to my branch
- [X] I've done self review and my code looks great!
- [X] I've tested my changes and written tests when applicable

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -1,5 +1,15 @@
 class Location < ActiveRecord::Base
   belongs_to :event
+
+  attr_accessor :addressStreet, :addressSuburb, :addressCity, :addressCountry
+
+  def address
+    [addressStreet, addressSuburb, addressCity,addressCountry].compact.join(', ')
```
Comment (Janther) [inline app/models/location.rb]: Space after comma missing.

File: app/views/locations/_location_fields.html.haml [hunk 2]
```diff
@@ -1,12 +1,18 @@
 .form-group
   = f.label :name
-  = f.text_field :name
+  = f.text_field :name, placeholder: 'The Beehive'
 .form-group
   = f.label :desription
-  = f.text_field :description
+  = f.text_field :description, placeholder: 'Ministerial offices.'
```
Comment (Janther) [inline app/views/locations/_location_fields.html.haml]: trailing space

File: app/models/location.rb (hunk 1 above)
Comment (Tinusw) [inline app/models/location.rb]: resolved

File: app/views/locations/_location_fields.html.haml (hunk 2 above)
Comment (Tinusw) [inline app/views/locations/_location_fields.html.haml]: resolved!

**Pull request merged**